network_ips.go

```diff
@@ -88,3 +97,14 @@ func (c *Client) ShareIPAddresses(ctx context.Context, opts IPAddressesShareOpti
 	_, err := doPOSTRequest[InstanceIP](ctx, c, e, opts)
 	return err
 }
+
+// AllocateReserveIP allocates a new IPv4 address to the Account, with the option to reserve it
+// and optionally assign it to a Linode.
+func (c *Client) AllocateReserveIP(ctx context.Context, opts LinodeReserveIPOptions) (*InstanceIP, error) {
+	e := "networking/ips"
```

Comment:
This isn't quite related to this PR but I'm curious about what the difference between `POST networking/reserved/ips` and `POST networking/ips` (with reserved == `true`) is. Is this effectively just an alias?

Reply:
That is a good question! We are currently supporting creation of reserved IP's through both `POST networking/reserved/ips` and `POST networking/ips (with reserved == true)`. The only difference is that along with creation we are also able to assign reserved IP's to linodes using `POST networking/ips (with reserved == true)`.

Reply:
Got it, thanks!